### PR TITLE
Sarahsimionescu/tech 24 sponsors

### PR DESCRIPTION
**Pull request by sarahsimionescu**

### CODE REVIEW

**Comment by sarahsimionescu** [comment]
WHaa

**Comment by sarahsimionescu** [comment]
plz help @err53 @Krish120003

**Comment by err53** [comment]
@sarahsimionescu added sharp so it builds, still gotta fix file formatting (also can we use a placeholder that's not so undersized?)

**Comment by err53** [comment]
or even just don't put placeholders for now idk

**Comment by sarahsimionescu** [comment]
Sure! How can I satisfy the rest of the checks?

**Comment by err53** [comment]
> Sure! How can I satisfy the rest of the checks?

1. Looks like `src/components/Sponsors.astro` isn't properly formatted, run the VSCode format code command on it
2. For the lighthouse lint, there's not much we can do right now, since we don't have all the sponsorship images. I'd recommend commenting out the placeholders for now, since it clearly works in the preview, and we'll need to change spacing based on the final received images anyways.

**Comment by err53** [comment]
Oh yeah, feel free to remove the colour override in for the bg, we plan to cover that in global styling instead (or merge the changes from the base theme branch anyways

**Comment by sarahsimionescu** [comment]
1. fixed
2. also fixed
Let me know if there is anything else I should do! @err53

**Comment by err53** [comment]
Ok, 2 more things that should be fixed before merging:

3. Currently we're not resizing images for different screen sizes which is not optimal, I just pushed some code that should enable that, please implement this for all other images. Use this tool to find the right sizes string for each image: https://ausi.github.io/respimagelint/
4. We're currently hard-coding each row, which isn't the best solution. Would it be possible to use a single 12-column wide grid, use col-span for each element in your array, and then use a for loop to iterate through each element in the list? This should significantly improve the readability and maintainability of the code. See: https://tailwindcss.com/docs/grid-column

File: src/layouts/main.astro [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-import "@fontsource-variable/inter";
```
Comment (err53) [inline src/layouts/main.astro:2]: bro you can't just do this without letting us know, design team specified Inter as the font for a reason

File: src/components/Sponsors.astro [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+<div class="container flex max-w-3xl flex-col items-center text-[#2D9AEA]">
+  <h1 class="font-display mb-10 text-center text-6xl font-bold">
+    <span class="text-fill-transparent text-stroke-white text-stroke-2"
+      >MEET OUR</span
+    >
+     SPONSORS
+  </h1>
+
+  <!-- Grid container -->
+  <div class="flex w-full flex-col gap-y-5 pl-5 pr-5">
```
Comment (err53) [inline src/components/Sponsors.astro]: This is not clean at all, but it will do the job for now...

**Comment by err53** [comment]
@sarahsimionescu fixed your grid code to work programatically

**Review by err53 — APPROVED** [review]

**Comment by err53** [comment]
### Merge Activity

* **Oct 2, 12:54 PM**: @err53 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/deltahacks/landing-10/3).

**Pull request merged**